composer.json

```diff
@@ -25,12 +25,13 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ^8.0",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "~1.0.0",
-        "phpunit/phpunit": "^5.7.27 || ^6.5.8 || ^7.1.4"
+        "phpunit/phpunit": "^9.4",
```

Comment:
By bumping phpunit, you definitely have to change almost all tests contain `setUp` and `tearDown` aswell as have to check for some assertion methods.

Did you execute tests locally? You might want to fix those tests even without travis. 👍

Reply:
I did run the tests on PHP 8 and via `./vendor/bin/phpunit` so not sure what happened there if they would be incompatible. I will double-check and see if I did something wrong 👍

Reply:
@boesing There are two test cases in the suite, and neither use `setUp` or `tearDown`, so tests actually _do_ run!

@xvilo I'm doing updates and testing manually, and will push to your branch for final review shortly. Thanks for getting this started!